lib/user/pages/pre_login/pre_login.dart

```diff
@@ -1,21 +1,22 @@
 import 'package:flutter/material.dart';
+import 'widgets/CarouselWidget.dart';
+import 'widgets/ButtonGroupWidget.dart';
 
 class PreLogin extends StatelessWidget {
   // This widget is the root of your application.
   @override
   Widget build(BuildContext context) {
     return Scaffold(
       body: Column(
-        children: <Widget>[TextField()],
+        children: <Widget>[
+          CarouselWidget(),
+          Expanded(child: ButtonGroupWidget()),
+          Container(
```

Comment:
Kan du ikke gemme denne container uden for din scaffold i en final variabel

Reply:
Nej